### PR TITLE
provides recheck button on verification screen

### PR DESCRIPTION
**Pull request by jessopb**
After receiving verification, user can click the "check again" button to continue doing what they were doing instead of reloading the app.

### CODE REVIEW

File: src/ui/component/userVerify/view.jsx [hunk 1]
```diff
@@ -25,15 +26,16 @@ class UserVerify extends React.PureComponent<Props> {
...
     return (
       <React.Fragment>
         <section className="section__header">
           <h1 className="section__title--large">{__('Extra Verification Needed')}</h1>
           <p>
             {__(
-              "We weren't able to auto-approve you for rewards. Please complete one of the steps below to unlock them."
+              "We weren't able to auto-approve you for rewards. Please complete one of the steps below to unlock them. Then you can"
             )}{' '}
+            <Button onClick={() => fetchUser()} button="link" label={__('check again.')} /> {'Or'}{' '}
```
Comment (seanyesmunt) [inline src/ui/component/userVerify/view.jsx]: I think this can just say "Refresh". Then we don't need the change to the other text.

**Comment by tzarebczan** [comment]
We should add it to this screen too: https://github.com/lbryio/lbry-desktop/blob/master/src/ui/page/rewards/view.jsx#L57

**Comment by jessopb** [comment]
Added Tom's change.
I find the bottom table jumping up on refresh jarring now, though.

**Comment by seanyesmunt** [comment]
@jessopb I'd be fine with that. I've noticed it can be a little jumpy so anything to improve that would be great

**Pull request merged**